src/core/errorHandler.js

```diff
@@ -45,7 +45,7 @@ const errorHandler = {
     /**
      * Get a context by it's name and create it if it doesn't exists
      * @param {String} name - the context name
-     * @returns {Object} the handling context
+     * @returns {Object|undefined} the handling context
      */
```

Comment:
i understand this task is just fixing the doc, so no issues, but wouldnt it be nice that this handler return null instead of undefined? :)

Reply:
Yes, actually it would be better to refactor the code in order to keep a consistent behavior.

But I did not want to change it, and only avoid adding lint error due to the version change